navigator/backend/src/main/scala/com/digitalasset/navigator/model/package.scala

```diff
@@ -15,6 +15,7 @@ package object model {
   /**
     * An opaque identifier used for templates.
     * Templates are usually identified using a composite type (see [[DamlLfDefRef]]).
+    * FIXME the reference to [[DamlLfDefRef]] is broken, where should it point?
```

Comment:
Should probably be `[[DamlLfIdentifier]]`

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1950/commits/f9221048bb342e3e9e8303c8ed7b8a7fb8405575